src/serial/private/utils/list_serialports_windows.nim

```diff
@@ -71,9 +91,12 @@ iterator listSerialPorts*(): string =
 
         if regKey != cast[HKEY](-1):
           # Then read the port name from the registry
-          yield getUnicodeValue("", "PortName", regKey)
+          var portname = getUnicodeValue("", "PortName", regKey)
```

Comment:
I normally prefer `let` over `var` where possible. Not a big deal though.

Reply:
1) It's your project so 'let' is what I will use. :satisfied:  2) I happen to agree with you that whenever he var is not changed 'let' is semantically a better option. :thumbsup:  I've fixed it in the latest commit.